openwisp_utils/admin_theme/menu.py

```diff
@@ -98,9 +98,15 @@ def __init__(self, config):
             raise ImproperlyConfigured(
                 f'"url" should be a type of "str". Error for the config- {config}'
             )
-        self.url = url
+        self.url = self.make_django_url(url)
         self.icon = config.get('icon')
 
+    def make_django_url(self, url):
+        # Add "/" at the end of link
+        if url[-1] != '/':
+            return url + '/'
+        return url
```

Comment:
not sure why this is needed, can you explain? May be good to add a comment.

Reply:
That is need because we compare url for links and current URL of page to find active link